release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -56,6 +56,70 @@ link:https://access.redhat.com/support/policy/updates/openshift[Red Hat OpenShif
 
+[id="ocp-4-6-web-console"]
+=== Web console
+
+[id="ocp-4-6-web-console-improved-upgrade-experience"]
+==== Improved upgrade experience in the web console
+
+* Administrators are now better informed about the differences between the upgrade
+channels via helpful text and links in the web console.
+* Changes such as bug fixes and enhancements are now included in each minor and
+patch release.
+* There is now a visual representation of the different upgrade paths.
+* Special Alerts now inform administrators when new patch releases, new minor
+releases, and news channels become available.
+
+[id="ocp-4-6-web-console-improved-operator-installation-workflow"]
+==== Improved Operator installation workflow with OperatorHub
+
+When administrators install Operators with OperatorHub, they now get immediate
+feedback to ensure that the Operator is installing properly.
+
+[id="ocp-4-6-web-console-improved-operand-details-view"]
+==== Improved Operand details view
+
+You can now see the schema grouping of `specDescriptor` fields and the status of
+your Operands on the Operand's details view, so that you can easily see the
+status and configure the `spec` of the Operand instance.
+
+[id="ocp-4-6-web-console-view-operators-related-objects"]
+==== View cluster Operators' related objects
+
+Previously, when viewing a cluster Operator, it was not clear what resources the
+Operator was associated with. When troubleshooting a cluster Operator, it could
+be challenging to locate the logs for all the resources that the Operator
+managed, which might be needed for troubleshooting. Now, with {product-title}
+{product-version}, you can expose a list of related objects of a Cluster
+Operator and easily review one of the related objects' details or YAML code view
+for troubleshooting.
+
+[id="ocp-4-6-web-console-warning-messages-when-editing-managed-resources"]
+==== Warning messages when editing managed resources
+
+Some resources are managed, such an Operator managed by a deployment, route,
+service, or ConfigMap. Users are discouraged from editing these resources.
+Instead, users should edit the custom resources for the Operator and its
+operand, and expect the Operator to update its related resources. With this
+update:
+
+* A *Managed by* label now appears to the right of the resource name with a
+clickable resource link for the managing resource.
+* When the resource is modified or deleted, a message appears warning the user
+that their changes might be reverted.
+* An Operator developer can now specify a custom message to appear via an
+annotation, guiding the user on what resource the user should edit.
+
+[id="ocp-4-6-web-console-specDescriptor-supports-CRD-instance"]
+==== `k8sResourcePrefix` specDescriptor supports CRD instance
+
+Operator authors, maintainers, and providers can now specify the prerequisite
+object for an Operand's instance with `Group/Version/Kind` for assigning a
+resource type other than Kubernetes core API.
+
+For more information, see
```

Comment:
@sferich888 Are you okay with me including this link?

Reply:
@sferich888  https://issues.redhat.com/browse/OSDOCS-1475 addresses that; we can update this link once that work is complete. Please let me know if you have any concerns with that. Thanks!